docs/Users_Guide/tc-diag.rst

```diff
@@ -226,43 +237,38 @@ If true, all input fields are read efficiently from each file in a single call.
 
 .. code-block:: none
 
-  nc_rng_azi_flag = TRUE;
-  nc_diag_flag    = FALSE;
-  cira_diag_flag  = FALSE;
-
-These three flag entries are booleans specifying what output data types should be written. The **nc_rng_azi_flag** entry controls the writing of a NetCDF file containing the cylindrical coordinate range-azimuth data used to compute the diagnostics. The **nc_diag_file** entry controls the writing of the computed diagnostics to a NetCDF file. The **cira_diag_flage** entry controls the writing of the computed diagnostics to a formatted ASCII output file. At least one of these flags must be set to true.
+  nc_cyl_grid_flag = TRUE; // ends with "_cyl_grid_{domain}.nc"
```

Comment:
Could change the comment to say: 'resulting output file ends with "_cyl_grid_{domain}.nc' and likewise for the following two lines.

Reply:
Or just remove the comments, since it's explained in the section below.

Reply:
Made the change on my laptop and will push with a bundle of changes in a single commit.

Reply:
It seems that this note should be removed:  I think it's line 29 / 20 (in changed version) of tc_diag.rst. Not sure why I can't comment on that section. 

![image](https://github.com/dtcenter/MET/assets/19677014/04c25e96-32de-4682-be65-fea53cbc0c01)